.scripts/update_readme_status.py

```
@@ -0,0 +1,105 @@
+
+        # Check if version contains beta indicators
+        is_beta = bool(re.search(r'(b\d*|beta|alpha|rc|dev)', version.lower()))
+
+        return {
+            "name": name,
+            "path": os.path.basename(addon_path),
+            "version": version,
+            "is_beta": is_beta,
+            "status": "⚠️" if is_beta else "✅"
+        }
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)